addons/hr_skills/static/src/scss/report_employee_cv.scss

```diff
@@ -80,140 +63,57 @@ div.o_employee_cv {
         .o_sidebar_section {
             padding: 15px;
         }
-
-        .o_sidebar_title {
-            text-transform: uppercase;
-            font-size: 16px;
-            font-weight: bold;
-            margin-top: 0;
-            margin-bottom: 15px;
-        }
-
-        .o_sidebar_education {
-            .o_sidebar_education_degree {
-                font-size: 14px;
-                margin-top: 0;
-                margin-bottom: 0;
-            }
-
-            .o_sidebar_education_line {
-                margin-bottom: 15px;
-                overflow-wrap: break-word;
-                &:last-child {
-                    margin-bottom: 0;
-                }
-            }
-
-            .o_sidebar_education_description {
-                color: rgba(256, 256, 256, 0.6);
-                font-weight: bold;
-                margin-bottom: 0px;
-                margin-top: 0;
-                font-size: 14px;
-            }
-
-            .o_sidebar_education_description_year {
-                color: rgba(256, 256, 256, 0.6);
-                font-weight: bold;
-                margin-bottom: 0px;
-            }
-        }
-
-        .o_sidebar_language {
-            .o_sidebar_language_level {
-                color: rgba(256, 256, 256, 0.6);
-            }
-        }
-            
-        .o_sidebar_list {
-            margin-bottom: 0;
-            li {
-                margin-bottom: 10px;
-                &:last-child {
-                    margin-bottom: 0;
-                }
-            }
-        }
     }
 
     .o_company {
         width: inherit;
-        display: inline-block;
         max-width: 49%;
         min-width: 49%;
         box-sizing: border-box;
-        vertical-align: top;
-        text-align: center;
     }
 
     .o_main_panel {
-        background: #fff;
+        background: $white;
         max-width: 100%;
```

Comment:
`w-100`?

Same for the rest of the file, I think you can change most of custom margin/padding/size with BS classes. 
The values won't be exactly the same, but the final render won't change much, but we would be able to remove a lot of custom css.